### ISSUE TITLE
Limit ability to kick users from video sesssion

### ISSUE THREAD

**Issue by GeekOnCoffee** [ThatConference/that.us#111]
Right now, anybody can kick anybody

**Comment by brettski** [comment]
@GeekOnCoffee why would we want to limit users from being able to self moderate their sessions? If a bad actor joins a session that person needs to be dealt with. Yes, I guess a bad actor can come in and start kicking everyone as well. Perhaps as a future feature enable based on a 'reputation' score or similar?

**Comment by GeekOnCoffee** [comment]
Yeah, we need people with the ability to moderate, but it probably shouldn't be global... 

Maybe: 
Crew
Session Creator
First to Join


Could potentially even be something like "Everybody who paid for a ticket"
